tests/python/conftest.py

```
@@ -0,0 +1,10 @@
+"""pytest 共通設定．scripts/ をモジュール検索パスに追加する．"""
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `．` (FULLWIDTH FULL STOP). Did you mean `.` (FULL STOP)?

(RUF002)